### PR TITLE
Fix disabled unit tests

### PR DESCRIPTION
**Pull request by roelandkulk**

### CODE REVIEW

File: src/test/java/org/littil/api/exception/ThrowableMapperTest.java [hunk 1]
```diff
@@ -16,13 +18,16 @@
 
 @QuarkusTest
 @TestHTTPEndpoint(GuestTeacherResource.class)
```
Comment (MarcelWildenburg) [inline src/test/java/org/littil/api/exception/ThrowableMapperTest.java:20]: @pschildkamp : should we use **@QuarkusTestResource(APIManagementMock.class)** here ?

File: src/test/java/org/littil/api/guestTeacher/service/GuestTeacherServiceTest.java [hunk 2]
```diff
@@ -147,32 +150,27 @@ void givenSaveTeacherUnknownErrorOccurred_thenShouldThrowPersistenceException()
...
-        entity.setId(teacherId);
-        entity.setSurname(surname);
-        entity.setFirstName(firstName);
+        final GuestTeacher guestTeacher = createGuestTeacher(null, firstName, surname, address, postalCode, locale);
+        final GuestTeacherEntity entity = createGuestTeacherEntity(teacherId, firstName, surname);
 
         final GuestTeacher expectedGuestTeacher = new GuestTeacher();
         expectedGuestTeacher.setId(entity.getId());
         expectedGuestTeacher.setSurname(entity.getSurname());
         expectedGuestTeacher.setFirstName(entity.getFirstName());
```
Comment (MarcelWildenburg) [inline src/test/java/org/littil/api/guestTeacher/service/GuestTeacherServiceTest.java:163]: ExpectedQuestTeacher is not used, so these lines can be removed.

**Review by MarcelWildenburg — COMMENTED** [review]
I have added some comments.
The saveTeacher method is only tested for PersistenceException. 
Maybey you can add a test for 
- Happy flow
- saveTeacher for unknown user

File: src/test/java/org/littil/api/guestTeacher/service/GuestTeacherServiceTest.java (hunk 2 above)
Comment (roelandkulk) [reply src/test/java/org/littil/api/guestTeacher/service/GuestTeacherServiceTest.java:173]: Thanks, this has been removed now.

**Comment by roelandkulk** [comment]
> I have added some comments. The saveTeacher method is only tested for PersistenceException. Maybey you can add a test for
> 
>     * Happy flow
> 
>     * saveTeacher for unknown user

These have been added.

**Comment by roelandkulk** [comment]
I have refactored GuestTeacherResourceTest.
Only givenDeleteTeacherById_thenShouldDeleteSuccessfully() is not working. The delete request is caught by AbstractSecurityInterceptor, tokenHelper.getCustomClaim() returns null. Unfortunately it's not possible to mock this.

I have removed @NotEmpty(message = "{GuestTeacher.locale.required}") from GuestTeacherPostResource and GuestTeacher since providing the locale is optional. There is still some work needed to map locale to location.country and back when a locale is provided but I think this is more suitable under a new issue / branch.

**Review by pschildkamp — APPROVED** [review]

**Pull request merged**